wp-seo-main.php

```diff
@@ -11,7 +11,7 @@
 }
 
 /**
- * @internal Nobody should be able to overrule the real version number as this can cause serious issues
+ * @note Nobody should be able to overrule the real version number as this can cause serious issues
```

Comment:
Unfortunately, PHPStorm strikes through the used constant or function if an `@internal` comment is added.